exercises/sum.cpp

```diff
@@ -6,3 +6,19 @@
   Insert the second number: 2
   Sum: 3
 */
+
+#include <iostream>
+
+void main() {
+  int a, b, sum;
+  
+  cout<<'Inserire un numero';
+  cin>>a;
+  cout<<'Inserire secondo numero';
```

Comment:
In c++ le stringhe possono essere indicate solo con le doppie virgolette. Le singole virgolette indicano un carattere

```c++
char c = 'c';
string s = "string";
```